hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -545,7 +545,7 @@ private Constants() {
   public static final String S3GUARD_METASTORE_LOCAL_ENTRY_TTL =
       "fs.s3a.s3guard.local.ttl";
   public static final int DEFAULT_S3GUARD_METASTORE_LOCAL_ENTRY_TTL
-      = 10 * 1000;
+      = 120 * 1000;
```

Comment:
seems to be a common problem, this was increased to 60s in https://github.com/apache/hadoop/pull/624 / https://github.com/apache/hadoop/pull/630